src/components/UserProfile/index.jsx

```diff
@@ -1,49 +1,79 @@
-import * as React from "react";
+import React, { Fragment } from "react";
+import { Link } from "react-router-dom"
 import * as Cards from "../VoyageCard/VoyageCard";
 import UserSideBar from "./UserSideBar";
 import Request from "../utilities/Request"
 import profileQuery from "./graphql/profileQuery"
 import './UserProfile.css'
+import dateFormatter from "../utilities/dateFormatter.js"
 
-class UserProfile extends React.Component {
-  state = {
-    user: this.props.data.user,
-    currentTeams: [],
-    pastTeams: [],
-    pendingApproval: [],
-    editable: false
-  }
 
-  updateState = () => {
-    let { data: { user }, editable } = this.props;
-    this.setState({ user: user, editable: editable ? true : false }, () => {
-      // TODO: Check filters
-      let { user } = this.state;
-      let { teams, cohorts } = user;
-      let currentTeams = teams.filter(team => { return team.cohort.status === 'ongoing' });
-      let pastTeams = teams.filter(team => { return team.cohort.status === 'ended' });
-
-      let pendingApproval = cohorts.filter((cohort) => {
-        let member = cohort.members.filter((member) => member.user.username === user.username && member.status === 'pending_approval');
-        if (member.length >= 1) {
-          return cohort;
-        }
-      });
-      this.setState({ currentTeams, pastTeams, pendingApproval });
-    })
-  }
-  componentDidMount() {
-    this.updateState();
-  }
+// PROJECT CARD INFO //
+const InfoComponents = ({ team }) => {
+  const { project, cohort, tier, title } = team
+  const infoObjects = [
+    { label: 'Voyage Dates', data: dateFormatter(cohort.start_date) + " - " + dateFormatter(cohort.end_date) },
+    { label: 'Team Name', data: title },
+    { label: 'Project', data: project.title },
+    { label: 'Elevator Pitch', data: project.elevator_pitch },
+    { label: 'Tier', data: 'Tier ' + tier.level },
+    { label: 'Team', data: project.users },
+    // { label: 'Status', data: cohort.status },
+    // { label: 'TechStack', data: project.skills },
+  ]
 
-  componentDidUpdate(prevProps) {
-    if (this.props !== prevProps) {
-      this.updateState();
+  return infoObjects.map((info, idx) => {
+    let data;
+    switch (info.label) {
+      case 'Team':
+        data = info.data.map((user, idx) => {
+          return (
+            <Link to={`/profile/${user.username}`} key={idx} className="team-card-user">
+              <img className="team-card-avatar-img" src={user.avatar ? user.avatar : require('../../assets/blank image.png')} alt={user.username} />
+              {/* <div className="team-card-username">{user.username}</div> */}
+            </Link>
+          )
+        })
+        break;
+      case 'TechStack':
+        data = info.data && info.data.map((tech, idx) => {
+          return (
+            <div key={idx} className="team-card-techstac k">{tech.name}</div>
```

Comment:
does this className need to be fixed?

Reply:
Yeah. Not used atm, so never saw the issue.